### PR TITLE
Staging merge fixes

### PR DESCRIPTION
**Pull request by samsaksfithian**
Stuff to do, in addition to @glenflorendo 's general comments on the previous pull request:

- [x]  Perform the requested changes
- [x] Prune unused dependencies / dev dependencies
- [x] Remove console statements
- [x] Simplify imports
- [ ] Document code
- [x] Expand linter to include more rules in order to remove eslint comments

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ const Navbar = ({ user, handleLogout }) => (
         <div>
           <li>
             <NavLink to="/account">
-              <FontAwesomeIcon icon="book" />
+              <FontAwesomeIcon icon="user" />
```
Comment (iylim) [inline src/components/Navbar.js]: yay!

File: src/pages/AboutPage.js [hunk 2]
```diff
@@ -80,8 +73,8 @@ const AboutPage = () => (
             <img src="/images/sam.png" alt="Sam Saks-Fithian" className="avatar-image" />
           </div>
           <h3 className="avatar-name">Sam</h3>
-          <h5 className="avatar-role">Software Engineer</h5>
-          <p className="avatar-movie">Favorite Movie: Black Panther </p>
+          <h5 className="avatar-role">GitHub Manager / Software Engineer</h5>
+          <p className="avatar-movie">Favorite Movies: MCU, especially Black Panther</p>
```
Comment (iylim) [inline src/pages/AboutPage.js]: we have to all say movie to be consistent.

Comment (iylim) [reply src/pages/AboutPage.js]: will allow Marvel if you add designer to Kevin?
add assistant to product manager for Glen?

File: src/pages/FindMoviePage.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const FindMoviePage = () => (
```
Comment (iylim) [inline src/pages/FindMoviePage.js]: Find Yamovie Page TY

File: src/App.js [hunk 4]
```diff
@@ -2,16 +2,15 @@ import React, { Component } from 'react';
...
-import ChatWindow from './components/ChatWindow';
-import Login from './components/Login';
-import Signup from './components/Signup';
+import {
+  HomePage,
+  BrowsePage,
+  AboutPage,
+  FindMoviePage,
+  NotFoundPage,
+} from './pages/Pages';
```
Comment (glenflorendo) [inline src/App.js]: If you rename your `Pages.js` file to `index.js`, then you can import like `from ./pages` without specifying the `index.js`.

File: src/App.js [hunk 5]
```diff
@@ -2,16 +2,15 @@ import React, { Component } from 'react';
...
+import { ChatWindow, Login, Signup, LloydChat, Navbar } from './components/Components';
```
Comment (glenflorendo) [inline src/App.js]: If you rename your `Components.js` file to `index.js`, then you can import like `from ./components` without specifying the `index.js`.

File: src/components/Components.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
+import ChatWindow from './ChatWindow';
```
Comment (glenflorendo) [inline src/components/Components.js]:
```suggestion
export { default as ChatWindow } from "./ChatWindow";
export { default as GenreList } from "./GenreList";
export { default as GoogleLogin } from "./GoogleLogin";
...
```

File: src/components/ChatWindow.js [hunk 7]
```diff
@@ -1,3 +1,4 @@
+// TODO: Remove this disable once Will's changes are merged in so the state is actually used
```
Comment (glenflorendo) [inline src/components/ChatWindow.js]: Let's try to keep this in mind.

File: src/pages/Pages.js [hunk 8]
```diff
@@ -0,0 +1,7 @@
+import AboutPage from './AboutPage';
```
Comment (glenflorendo) [inline src/pages/Pages.js]:
```suggestion
export { default as AboutPage } from "./AboutPage";
export { default as BrowsePage } from "./BrowsePage";
export { default as HomePage } from "./Homepage";
// Continue with the rest
```

File: src/components/Components.js (hunk 6 above)
Comment (samsaksfithian) [reply src/components/Components.js]: Oh nice, I didn't know you could batch an import and an export together

File: src/components/ChatWindow.js (hunk 7 above)
Comment (samsaksfithian) [reply src/components/ChatWindow.js]: Just install that TODO highlighter extension thing to make sure it's very visible :P

**Comment by glenflorendo** [comment]
Thank you for your hard work, everyone!

**Pull request merged**